site/content/usage/11-customizing-the-kubelet.md

```diff
@@ -45,7 +45,14 @@ nodeGroups:
         evictionHard:
             memory.available:  "200Mi"
             nodefs.available: "10%"
+        featureGates:
+            DynamicKubeletConfig: true
```

Comment:
Could you please add `RotateKubeletServerCertificate` here as well, and make sure to add a comment, e.g.

```
                         RotateKubeletServerCertificate: true # has to be enabled, otherwise it will be disabled
```